diff --git a/CMakeLists.txt b/CMakeLists.txt
index b3f12a4..d25193e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,6 +8,14 @@ SET(PROJECT_VER_PATCH 0)
 SET(PROJECT_VER "${PROJECT_VER_MAJOR}.${PROJECT_VER_MINOR}.${PROJECT_VER_PATCH}")
 SET(PROJECT_APIVER "${PROJECT_VER}")
 
+if(ARM64)
+   set(BUILD_MMAL FALSE)
+   set(BUILD_MMAL_APPS FALSE)
+else()
+   set(BUILD_MMAL TRUE)
+   set(BUILD_MMAL_APPS TRUE)
+endif()
+
 set(vmcs_root ${PROJECT_SOURCE_DIR})
 get_filename_component(VIDEOCORE_ROOT . ABSOLUTE)
 
@@ -68,10 +76,15 @@ add_subdirectory(interface/vmcs_host)
 add_subdirectory(interface/vchiq_arm)
 
 #add_subdirectory(opensrc/tools/lua)
+if(BUILD_MMAL)
+   include_directories(interface/mmal)
+   add_subdirectory(interface/mmal)
+   add_subdirectory(containers)
+endif()
 
 # VidTex supports Android and Linux
 if(BUILD_MMAL_APPS)
-add_subdirectory(host_applications/android/apps/vidtex)
+#add_subdirectory(host_applications/android/apps/vidtex)
 endif(BUILD_MMAL_APPS)
 
 if(NOT ARM64)
diff --git a/host_applications/linux/CMakeLists.txt b/host_applications/linux/CMakeLists.txt
index f00a112..b564593 100644
--- a/host_applications/linux/CMakeLists.txt
+++ b/host_applications/linux/CMakeLists.txt
@@ -1,3 +1,5 @@
 # linux apps
 
 add_subdirectory(libs/bcm_host)
+add_subdirectory(libs/sm)
+add_subdirectory(libs/debug_sym)
